styles/themes/teammember/normalize.scss

```diff
@@ -0,0 +1,186 @@
+/* http://meyerweb.com/eric/tools/css/reset/  v2.0b1 | 201101  */
```

Comment:
But I agree that we should stick to one or the other and since there's already one referenced in site.scss we can use that.